dnf-testing.sh

```diff
@@ -1,196 +1,12 @@
-#!/bin/bash
-set -euo pipefail
+#!/usr/bin/bash
```

Comment:
Hey, this actually doesn't work for me :slightly_smiling_face: I have `/bin/bash` but no `/usr/bin/bash`...

Reply:
Given that there is /bin -> /usr/bin symlink on Fedora/RHEL I changed the shebang to /bin/bash.